keyboards/ergotravel/keymaps/via/keymap.c

```diff
@@ -0,0 +1,55 @@
+     KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_SPC,           KC_SPC,  KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
+  //|--------+--------+--------+--------+--------+--------+--------|        |--------+--------+--------+--------+--------+--------+--------|
+     KC_LCTL, KC_LGUI, KC_LALT, _______,           _______,   KC_SPC,           KC_SPC,  _______,            KC_LEFT, KC_UP,   KC_DOWN, KC_RGHT
+  //`--------+--------+--------+--------+--------+--------+--------/        \--------+--------+--------+--------+--------+--------+--------'
+  ),
+
+  [_LOWER] = LAYOUT(
+  _______,  _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______,           _______, _______,         _______,  _______,          _______, _______, _______, _______
+
+  ),
+
+  [_RAISE] = LAYOUT(
+ 
+_______,  _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______,           _______, _______,         _______,  _______,          _______, _______, _______, _______
+  
+  ),
+
+  [_ADJUST] = LAYOUT(
+     _______,  _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______,           _______, _______,         _______,  _______,          _______, _______, _______, _______
+  ),
+  [_FINL] = LAYOUT(
+     _______,  _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______,  _______, _______, _______, _______, _______,         _______,  _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______,           _______, _______,         _______,  _______,          _______, _______, _______, _______
+  )
+
```

Comment:
One last thing, I think: this layer should be removed, or `DYNAMIC_KEYMAP_LAYER_COUNT` set to 5 in the via keymap config.h.